docs/UserGuide.md

```diff
@@ -69,31 +69,210 @@ The following annotated text-boxes are used throught this guide to provide usefu
-   > Some example commands you can try:
-   > * `list` : Lists all contacts.
-   > * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` tothe Address Book.
-   > * `delete 3` : Deletes the 3rd contact shown in the current list.
-   > * `clear` : Deletes all contacts.
-   > * `exit` : Exits the app.
-
-1. Refer to the [Features](#features) below for details of each command.
+# Quick Start
+
+In this section, lets learn how to set up MediBase3 on your computer.
+
+---
+
+## Installing MediBase3
+
+### Step 1: Install JDK 17
+
+Before running MediBase3, ensure that **Java 17** is installed on your system.
+
+### For Windows:
+
+1. Visit the [Oracle JDK 17 download page](https://www.oracle.com/java/technologies/javase-jdk17-downloads.html).
+2. Download the appropriate installer for your system (e.g., `Windows x64 Installer`).
+3. Run the installer and follow the setup instructions.
+4. Verify the installation by opening **Command Prompt** and typing:
+```
+   java -version
+   ```
+You should see something like:
+
+![javaVersionWindows.jpeg](images%2FjavaVersionWindows.jpeg)
+This means that your machine is running Java version 17.0.12, which means that Java 17 is installed.
+
+### For macOS:
+
+1. Install **Homebrew** if you haven’t already. Open **Terminal** and type:
+```
+   /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+   ```
+2. Install Java 17 by typing:
+```
+   brew install openjdk@17
+   ```
+3. Link the installed JDK:
+```
+   sudo ln -sfn $(brew --prefix openjdk@17)/libexec/openjdk.jdk /Library/Java/JavaVirtualMachines/openjdk-17.jdk
+   ```
+4. Verify the installation by typing:
+```
+   java -version
+   ```
+You should see something like this:
+
+![javaVersionMacOS.png](images%2FjavaVersionMacOS.png)
+This means that your machine is running Java version 17.0.11, which means that Java 17 is installed.
+
+
+### For Ubuntu/Debian:
+
+1. Open **Terminal**.
+2. Install Java 17 by typing:
+```
+   sudo apt install openjdk-17-jdk
+   ```
+3. Verify the installation by typing:
+```
+   java -version
+   ```
+   You should see something like this:
+
+![javaVersionLinux.jpg](images%2FjavaVersionLinux.jpg)
+This means that your machine is running Java version 17.0.11, which means that Java 17 is installed.
+
+---
+
+{: .alert .alert-info}
+:information_source: **Note**
+Depending on your system setup, you may need to install JavaFX separately. You can refer to the [OpenJFX installation guide](https://openjfx.io/openjfx-docs/#install-javafx) for detailed instructions.
+
+{: .alert .alert-info}
+:information_source: **Note**
+For users of other operating systems, as long as you have the following installed, you can use MediBase3:
+- Java/OpenJDK 17
+- JavaFX/OpenJFX 17
+
+{: .alert .alert-info}
+:information_source: **Note**
+For Linux users with Wayland compositors, ensure you install **OpenJFX 21** for better compatibility.
```

Comment:
use block quotes, else the subsequent lines do not get included in the alert box:

![image](https://github.com/user-attachments/assets/d285eef1-993a-4fae-a866-a9c3c8c79926)